src-electron/generator/helper-endpointconfig.js

```diff
@@ -418,7 +457,38 @@ function endpoint_reporting_config_defaults(options) {
         .map((m) => `ZAP_CLUSTER_MASK(${m.toUpperCase()})`)
         .join(' | ')
     }
-    ret += `  { ZAP_REPORT_DIRECTION(${r.direction}), ${r.endpoint}, ${r.clusterId}, ${r.attributeId}, ${mask}, ${r.mfgCode}, {{ ${r.minOrSource}, ${r.maxOrEndpoint}, ${r.reportableChangeOrTimeout} }} }, /* ${r.name} */ \\\n`
+    let orderTokens = order.split(',').map((x) => (x ? x.trim() : ''))
+    let items = []
+    orderTokens.forEach((tok) => {
+      switch (tok) {
+        case 'direction':
+          items.push(`ZAP_REPORT_DIRECTION(${r.direction})`)
+          break
+        case 'endpoint':
+          items.push(r.endpoint)
+          break
+        case 'clusterId':
+          items.push(r.clusterId)
+          break
+        case 'attributeId':
+          items.push(r.attributeId)
+          break
+        case 'mask':
+          items.push(mask)
+          break
+        case 'mfgCode':
+          items.push(r.mfgCode)
+          break
+        case 'minmax':
+          items.push(
+            `{{ ${r.minOrSource}, ${r.maxOrEndpoint}, ${r.reportableChangeOrTimeout} }}`
```

Comment:
What happens if the minmax struct changes?

Reply:
Good point. I added minmaxorder to the helper for this case.